yarn.lock

```diff
@@ -2825,7 +2825,24 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@metamask/selected-network-controller@^12.0.1, @metamask/selected-network-controller@workspace:packages/selected-network-controller":
+"@metamask/selected-network-controller@npm:^12.0.1":
```

Comment:
Hmm, this shouldn't be here. I suspect this is here because the `queued-request-controller` manifest hasn't been updated.

Try running `yarn constraints --fix` to ensure all packages reference the updated `@metamask/selected-network-controller` version. Then run `yarn` again to fix this.

Reply:
Oh good catch! Thanks @Gudahtt. Done here: https://github.com/MetaMask/core/pull/4260/commits/08363e722cdc18881c1a6f023dcbc4aebd2d2b6e